components/tab-tile/tab-tile.scss

```diff
@@ -12,26 +18,74 @@
 
 .msds-tab-tile {
   display: flex;
+  flex: 1;
   flex-direction: column;
   align-items: center;
-  min-width: 277px;
-  height: 180px;
-  border: 1px solid $gray-4;
-  background: transparent;
+  margin-right: 4px;
+  background: $white;
+  max-width: 371px;
+  border-top: 4px solid $white;
+  border-left: 1px solid $white;
+  border-right: 1px solid $white;
   @include msds-font('body-1');
   font-weight: bold;
+  text-align: center;
   color: $gray-9;
   cursor: pointer;
-  padding-top: 24px;
+  padding: 20px 16px 24px 16px;
+
   svg {
     width: 120px;
     height: 90px;
     margin-bottom: 20px;
   }
 
+  &__text {
+    max-width: 200px;
+  }
+
+  &:hover {
+    color: $clear-blue;
+  }
+
+  &:focus {
+    outline: 4px solid rgba(0, 153, 218, 0.5);
```

Comment:
This rgba(0, 153, 218, 0.5) has been showing up throughout the design system, maybe we should look into moving rgba colours into the _variables.scss file

Reply:
discuss with Martin what he thinks